test/helpers/kubectl.go

```diff
@@ -4378,7 +4378,7 @@ func (kub *Kubectl) CleanupCiliumComponents() {
 		wg sync.WaitGroup
 
 		resourcesToDelete = map[string]string{
-			"configmap":          "cilium-config hubble-ca-cert hubble-relay-config",
```

Comment:
Can we run a Jenkins CI test for this? I believe we can safely remove it, but since this function here is also used in the upgrade tests (which deploys Cilium v1.10, which afair still creates the `hubble-ca-cert` ConfigMAP), I think it's worth running to ensure we don't break CI via leftover artifacts.